OJP/OJP_Trips.xsd

```diff
@@ -824,6 +824,7 @@
 					<xs:documentation>A list of references to SITUATIONs.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="SituationFullRef" type="SituationFullRefStructure" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
@sgrossberndt This comes from somewhere... but is likely correct.

Reply:
No, it is not correct. `SituationFullRefs` replaced `SituationFullRef`

Reply:
https://github.com/VDVde/OJP/pull/268/files